src/Standards/Generic/Docs/CodeAnalysis/EmptyPHPStatementStandard.xml

```diff
@@ -0,0 +1,44 @@
+<documentation title="Empty PHP Statement">
+    <standard>
+    <![CDATA[
+    Empty PHP tags are not allowed.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: There is at least one statement inside the PHP tag pair.">
+        <![CDATA[
+<?php <em>echo 'Hello World';</em> ?>
+<?= <em>'Hello World';</em> ?>
+        ]]>
+        </code>
+        <code title="Invalid: There is no statement inside the PHP tag pair.">
+        <![CDATA[
+<?php <em>// some comment</em> ?>
```

Comment:
Uh.. oh.. did you actually check the example code with the sniff ? You may want to do so...

Reply:
I'm sorry, I'm testing all the example codes before creating the documentation but I don't know what happened here. I just fixed it.